fixtures/varytec/hero-wash-340fx-rgbw-zoom.json

```diff
@@ -26,6 +26,9 @@
     "dimensions": [255, 187, 368],
     "weight": 6.1,
     "power": 134,
+    "powerConnector": {
```

Comment:
You used `powerConnector` in all fixtures, but `powerConnectors` in the schema. I prefer the latter, so please update all fixtures.

Reply:
I agree `powerConnectors` is better. Working on this now